packages/common/src/HardhatConfig.ts

```diff
@@ -143,6 +143,17 @@ export function getHardhatConfig(
         url: getNodeUrl("arbitrum-rinkeby", true, 421611),
         accounts: { mnemonic },
       },
+      sx: {
```

Comment:
what is this chain? what name is `sx`?

Reply:
SX Network is the Sport-X sidechain; they're looking at using the OO to settle their markets.